openassessment/xblock/openassessmentblock.py

```diff
@@ -934,6 +937,12 @@ def parse_xml(cls, node, runtime, keys, id_generator):
         block.text_response_editor = config['text_response_editor']
         block.title = config['title']
         block.white_listed_file_types_string = config['white_listed_file_types']
+
+        # Deserialize and add tag data info to block if any
+        if hasattr(block, 'read_tags_from_node') and callable(block.read_tags_from_node):  # pylint: disable=no-member
+            # This comes from TaggedBlockMixin
+            block.read_tags_from_node(node)  # pylint: disable=no-member
```

Comment:
This is working as expected manual testing in the `edx-platform`, but in the tests here, it fails. I think it is because this block doesn't have the XmlMixin. As it is a new dependency, I'm unsure if I should add it here or find another way to test it.

Reply:
Yes, I faced similar issues when implementing the copy functionality. Since we don't have access to the `XBLOCK_MIXINS` in edx-platform in this repo (because edx-ora2 is imported into edx-platform), you need to manually add the methods/fields for the tests to work.

Here is an example of how I did it in my tests:
https://github.com/openedx/edx-ora2/blob/c76a89c4f9e99c5452bd4f7dc07c6002ae7a5564/openassessment/xblock/test/test_xml.py#L219-L251

Reply:
Thank you @yusuf-musleh! It took me some time but I figure it out with your help!

Could you take another look at it please?